docs/data_node/extract_node/tube.md

```diff
@@ -46,9 +46,9 @@ Flink SQL> CREATE TABLE tube_extract_node (
      ) WITH (
      'connector' = 'tubemq',
      'topic' = 'topicName',
-     'masterRpc' = 'rpcUrl', -- 127.0.0.1:8715
+     'master.rpc' = 'rpcUrl', -- 127.0.0.1:8715
      'format' = 'json',
-     'groupId' = 'groupName');
+     'group.id' = 'groupName');
```

Comment:
Does group.Id means InlongGroupId？

Reply:
No, It means TubeMQ's consumer group name.

Reply:
change to `group.name`?

Reply:
done, change `group.id` to `group.name`.  The relevant code change is: PR #9031 https://github.com/apache/inlong/pull/9031, Commit https://github.com/apache/inlong/pull/9031/commits/618216091a1281d5617c33a7a9e1553be476bee9